api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/convolutional/Conv1D.kt

```diff
@@ -57,42 +56,27 @@ private const val EXTRA_DIM = 1L
  * @since 0.3
  */
 public class Conv1D(
-    public val filters: Int = 32,
-    public val kernelSize: Int = 3,
```

Comment:
Looks like  kernel size is a stable term in Deep Learning for CNN

Reply:
We can't use `kernelSize` here since `kernelSize` is an array and we want to allow passing a single integer here.